tests/cli/test_cli.py

```diff
@@ -878,6 +912,51 @@ def test_upload4(ctx):
         os.unlink(archive)
 
 
+@TestModule.register('blob')
+def test_blob(ctx):
+    """Certain Blob Storage tests. Should only be called when
+    CODALAB_ALWAYS_USE_AZURE_BLOB_BETA is set to 1."""
+
+    def fetch_contents_blob_no_redirect(uuid):
+        """Fetch blob contents without redirecting to Blob Storage.
+        We don't redirect to Blob Storage, and Target-Type and X-CodaLab-Target-Size
+        should always be present on the response, if we don't pass in the
+        `support_redirect` parameter. In a future version, we will change this so that
+        this guarantee no longer exists (see comment at bundles.py:_fetch_bundle_contents_blob).
+        """
+        return ctx.client._make_request(
```

Comment:
maybe the `supports_redirect=0` parameter should be included to be explicit?

Reply:
No, I want to test to ensure that the default behavior is to set it to 0.